documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -124,6 +124,7 @@ Fixes from previous versions
 * The 6.3.1 patch release includes all fixes from the patch releases 6.2.21 and 6.2.22. :doc:`(6.2 Release Notes) </release-notes/release-notes-620>`
 * The 6.3.3 patch release includes all fixes from the patch release 6.2.23. :doc:`(6.2 Release Notes) </release-notes/release-notes-620>`
 * The 6.3.5 patch release includes all fixes from the patch releases 6.2.24 and 6.2.25. :doc:`(6.2 Release Notes) </release-notes/release-notes-620>`
+* The 6.3.10 patch release includes all fixes from the patch releases 6.2.26 and 6.2.27. :doc:`(6.2 Release Notes) </release-notes/release-notes-620>`
```

Comment:
Based on a quick look, I think 6.2.26 is included in 6.3.9. If that's accurate, case we should have a separate line for each of the two patches.

Reply:
Oops good catch. Updated.